lib/generators/i18n/active_record/templates/migration.rb.erb

```diff
@@ -1,4 +1,4 @@
-class Create<%= name.classify %> < ActiveRecord::Migration[<%= ActiveRecord::Migration.current_version.to_s %>]
+class Create<%= name.classify %>s < ActiveRecord::Migration[<%= ActiveRecord::Migration.current_version.to_s %>]
```

Comment:
I think I know a bit cleaner way
```suggestion
class <%= migration_class_name %> < ActiveRecord::Migration[<%= ActiveRecord::Migration.current_version %>]
```

And could you please also change this line https://github.com/svenfuchs/i18n-active_record/blob/master/lib/generators/i18n/active_record/install_generator.rb#L22
to
```ruby
migration_template 'migration.rb.erb', 'db/migrate/create_#{table_name}.rb'
```
otherwise `translations` will be hardcoded